packages/primitives/form/src/form-message.component.ts

```diff
@@ -0,0 +1,29 @@
+import { NgFor } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'rdx-form-message',
+    standalone: true,
+    imports: [NgFor],
```

Comment:
you don't need import this, is part of [core](https://v17.angular.io/api/core/for)
```suggestion
    imports: [],
```